src/UXClient/Components/LineChart/LineChart.ts

```diff
@@ -1583,17 +1594,11 @@ class LineChart extends TemporalXAxisComponent {
             var self = this;
             this.hasStackedButton = true;
             this.stackedButton = this.chartControlsPanel.append("button")
-                .style("left", "60px")
```

Comment:
Any reason why this left style was removed ( ensuring some one else's alignment does not break )